pithos/plugins/notify.py

```diff
@@ -45,6 +45,14 @@ def on_prepare(self):
         if 'action-icons' in caps:
             self.notification.set_hint('action-icons', GLib.Variant('b', True))
 
+        # GNOME Shell 3.20 or higher has bultin MPRIS functionality that makes
+        # persistent song notifications redundant.
+        min_version = Gtk.get_major_version() >= 3 and Gtk.get_minor_version() >= 20
```

Comment:
> Maybe there is some better way to access this

The server_info contains a version. But again that is the version of the server not necessarily the version of GNOME Shell.